app/routes/__index/domains/index.tsx

```diff
@@ -75,7 +78,17 @@ export const action = async ({ request }: ActionArgs) => {
 };
 
 export default function DomainsIndexRoute() {
+  const revalidator = useRevalidator();
   const domains = useTypedLoaderData<typeof loader>();
+  const pending = domains.some((domain) => domain.status === 'pending');
+
+  // Check to see if any change is pending, and if so, reload every 5s until finished
+  useInterval(
+    () => {
+      revalidator.revalidate();
+    },
+    pending ? 5_000 : null
+  );
 
   return (
     <Container maxW="contianer.xl">
```

Comment:
Not related to this PR, but we should file an issue to fix the spelling error in the value for `maxW`.